mlir/test/CAPI/mixr_full.c

```
@@ -194,7 +194,7 @@ static bool constructAndTraverseIr(MlirContext ctx) {
       mlirRockTuningSpaceCreate(module, RocmlirTuningParamSetKindFull);
   printf("Got tuning space,\n");
   unsigned fNum = mlirRockTuningGetNumParams(tuningSpace);
-  // CHECK: full set = 567
+  // CHECK: full set = 979
```

why is this increasing? this would make full tuning significantly slower

---

It is because of additional `kPack` values 16, 32

For many MFMAs they kPack = 32 will fail early